src/Containers/StoreHoursList/StoreHoursList.tsx

```diff
@@ -0,0 +1,529 @@
+                                setStoreHours({
+                                    ...storeHours,
+                                    [time[INITIAL_TIME_INDEX]]: e.target.value
+                                })
+                            }
+                        /> //do a from to check
+                    );
+                })}
+                <StyledHeading type='h6'>
+                    { constants.TITLES.SELECT_A_CATEGORY }
+                </StyledHeading>
+                <StyledSelect
+                    onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                        setAddStoreHoursCategory(e.target.value);
+                    }}
+                    value={addStoreHoursCategory}
+                >
+                    {Object.entries(allCategoriesWithHours).map((listAllCategories): React.ReactElement => {
+                        return ( 
+                            <option
+                                key={listAllCategories[ALL_CATEGORIES_INDEX]} 
+                                value={listAllCategories[ALL_CATEGORIES_TIMES].category}
+                            >
+                                { listAllCategories[ALL_CATEGORIES_TIMES].category }
+                            </option>
+                        );
+                    })}
+                </StyledSelect>
+                <CenteredButton 
+                    onClick={(): void => {
+                        setAddModalState(!addModalState);
+                        saveHours(addStoreHoursCategory);
+                    }}
+                >
+                    { constants.BUTTONS.ADD_HOURS } 
+                </CenteredButton> 
+            </StyledModal>
+        </StoreHoursContainer>
+    );
+};
+
+const StoreHoursContainer = styled.div`
+`;
+const ButtonsContainer = styled.div`
+    ${Mixins.flex('center')};
+    ${Mixins.media(
+        'phone',
+        `
+        ${Mixins.flex('column')};  
+    `,
+    )}
+`;
+const TextContainer = styled.div`
+    ${Mixins.flex('center')};
+    ${Mixins.flex('column')};  
+`;
+const StyledButton = styled(Button)`
+    margin: 10px;
+`;
+const CenteredButton = styled(Button)`
+    margin: 10px auto;
+`;
+const StyledModal = styled(Modal)`
+    max-height: 60%;
+    ${({ theme }): string => `
+        padding: ${theme.dimensions.padding.container};
+    `};
+`;
+const StyledHeading = styled(Heading)`
+    font-weight: bold;
+    flex-wrap: wrap;
+    text-align: center;
+    margin: 10px;
+`;
+const DaysDiv = styled.div`
+    margin: 5px;
+    ${Mixins.media(
+        'phone',
+        `
+        ${Mixins.flex('column')};  
+    `,
+    )}
+`;
+const StyledCheckbox = styled(Checkbox)`
+    margin: 5px;
+`;
+const StyledSelect = styled(Select)`
+    margin: 5px;
+`;
+const TagContainer = styled.div`
+    margin: auto;
+    ${({ theme }): string => `
+        padding: ${theme.dimensions.padding.container};
+    `};
+`;
+const StyledTag = styled(Tag)`
+    margin: 5px;
+`;
```

Comment:
3 things with exactly the same style implies there should be relationship? maybe they are just spacing for sections of a component?

Reply:
Hi Frederic, I did notice that as well. They are spacing for sections of the component. What would be the better way to make this code less redundant?

Reply:
Make a SC called Section, and change the type using as
```
<Section as={OWO}>
...
</Section>

...

const Section = styled.div`margin: 5px`;
```